.ci/deploy-docs.sh

```diff
@@ -17,8 +20,8 @@ cp ../build/docs/html/* .
 
 git init
 git branch -m main
-git config user.name "Travis CI"
-git config user.email "travis@harfbuzz.org"
+git config user.name "CI"
+git config user.email "ci@harfbuzz.org"
```

Comment:
Maybe change to `harfbuzz-admin@googlegroups.com` while at it?

Reply:
Done.